tests/python_package_test/test_dask.py

```diff
@@ -1253,5 +1257,9 @@ def test_sklearn_integration(estimator, check, client):
 # this test is separate because it takes a not-yet-constructed estimator
 @pytest.mark.parametrize("estimator", list(_tested_estimators()))
 def test_parameters_default_constructible(estimator):
-    name, Estimator = estimator.__class__.__name__, estimator.__class__
+    name = estimator.__class__.__name__
+    if sk_version > parse_version("0.23"):
```

Comment:
What about `0.23.1` and `0.23.2`?
```suggestion
    if sk_version >= parse_version("0.24"):
```

Reply:
Sure. It doesn't really matter, since in 0.23.x this test supported passing either a class or an instance